assets/Technology-Tag-Icons/icons.tsx

```diff
@@ -0,0 +1,120 @@
+export const OffshoreWindIcon = () => (
+  <svg
+    width="9"
+    height="11"
+    viewBox="0 0 9 11"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M1.86023 9.24538C2.12061 8.9049 2.66514 8.91764 2.94023 9.21183C3.3108 9.608 3.83306 9.99679 4.65948 9.99679C5.50023 9.99679 6.01457 9.68482 6.36476 9.29369C6.63683 8.98943 7.21721 8.94045 7.5025 9.3014C7.7357 9.5966 7.9908 9.83142 8.49419 9.93105C8.56569 9.94697 8.62737 9.98706 8.66613 10.0428C8.70489 10.0985 8.71766 10.1655 8.70174 10.2295C8.68581 10.2934 8.64244 10.3493 8.58085 10.3853C8.51926 10.4213 8.4443 10.4345 8.37193 10.4221C7.67721 10.2853 7.3157 9.94044 7.0391 9.59056C7.02099 9.56741 6.98816 9.55031 6.93872 9.55131C6.9137 9.5516 6.88904 9.5567 6.86644 9.56625C6.84384 9.57581 6.82383 9.58959 6.80778 9.60666C6.36401 10.1025 5.69344 10.5 4.65948 10.5C3.59419 10.5 2.9274 9.98606 2.50476 9.5332C2.49274 9.52045 2.47765 9.51027 2.4606 9.50341C2.44355 9.49656 2.42498 9.49321 2.40627 9.49362C2.39099 9.4931 2.37582 9.49603 2.36216 9.50213C2.34851 9.50823 2.33681 9.5173 2.32816 9.5285C2.04665 9.89649 1.68551 10.2766 0.947024 10.4221C0.910331 10.4303 0.872168 10.4319 0.83479 10.4267C0.797411 10.4215 0.761576 10.4098 0.729402 10.3921C0.697228 10.3744 0.669368 10.3512 0.647469 10.3238C0.625569 10.2963 0.610075 10.2653 0.601901 10.2325C0.593727 10.1996 0.59304 10.1657 0.59988 10.1326C0.606721 10.0995 0.620949 10.068 0.641725 10.0399C0.662501 10.0118 0.689403 9.98771 0.720841 9.96901C0.752278 9.95031 0.787614 9.9374 0.82476 9.93105C1.35759 9.82571 1.61268 9.56876 1.86023 9.24538Z"
+      fill="#4974E0"
```

Comment:
Then for all the fill and stroke colors, make sure to add them to the colors.ts global colors stylesheet if they are not there already!